### PR TITLE
pin py311

### LINKED ISSUE #32: `graphblas` a problem for Py312 on `osx-arm64`?
Seems like `graphblas` might be a problem for Py312 on `osx-arm64`?

```
(base) jgaboardi@Jamess-MacBook-Air envs % mamba env create -f py312_uscuni_simplification.yml 
Channels:
 - conda-forge
Platform: osx-arm64
Collecting package metadata (repodata.json): done
Solving environment: failed
Channels:
 - conda-forge
Platform: osx-arm64
Collecting package metadata (repodata.json): done
Solving environment: failed

LibMambaUnsatisfiableError: Encountered problems while solving:
  - nothing provides _python_rc needed by python-3.12.0rc3-rc3_h47c9636_1_cpython

Could not solve for environment specs
The following packages are incompatible
├─ graphblas-algorithms is installable with the potential options
│  ├─ graphblas-algorithms [2022.11.0|2022.12.0|...|2023.6.0] would require
│  │  └─ python-graphblas [>=2022.11.0 |>=2023.1.0 ], which requires
│  │     └─ python-suitesparse-graphblas [>=7.2.0.0,<7.4 |>=7.3.0.2,<7.4 |>=7.3.2.0,<7.4 |>=7.4.0.0,<7.5 ] with the potential options
│  │        ├─ python-suitesparse-graphblas [7.3.0.0|7.3.0.2|...|7.4.4.0] would require
│  │        │  └─ python_abi 3.10.* *_cp310, which can be installed;
│  │        ├─ python-suitesparse-graphblas [7.3.0.0|7.3.0.2|...|7.4.4.0] would require
│  │        │  └─ python_abi 3.8.* *_cp38, which can be installed;
│  │        ├─ python-suitesparse-graphblas [7.3.0.0|7.3.0.2|...|7.4.4.0] would require
│  │        │  └─ python_abi 3.9.* *_cp39, which can be installed;
│  │        └─ python-suitesparse-graphblas [7.3.0.2|7.3.2.0|...|7.4.4.0] would require
│  │           └─ python_abi 3.11.* *_cp311, which can be installed;
│  └─ graphblas-algorithms 2022.4.1 would require
│     └─ python-graphblas >=2022.4.2  with the potential options
│        ├─ python-graphblas [2022.10.0|2022.10.1|...|2023.7.0], which can be installed (as previously explained);
│        ├─ python-graphblas 2022.4.2 would require
│        │  └─ python-suitesparse-graphblas >=7.0.3.1,<7.1 , which does not exist (perhaps a missing channel);
│        ├─ python-graphblas [2022.5.0|2022.6.0|2022.6.1] would require
│        │  └─ python-suitesparse-graphblas >=7.1.0.0,<7.2 , which does not exist (perhaps a missing channel);
│        └─ python-graphblas 2022.9.0 would require
│           └─ python-suitesparse-graphblas >=7.2.0.0,<7.3 , which does not exist (perhaps a missing channel);
└─ python 3.12**  is not installable because there are no viable options
   ├─ python [3.12.0|3.12.1|3.12.2] would require
   │  └─ python_abi 3.12.* *_cp312, which conflicts with any installable versions previously reported;
   └─ python 3.12.0rc3 would require
      └─ _python_rc, which does not exist (perhaps a missing channel).
```

### PR DESCRIPTION
**Pull request by jGaboardi**
This PR resolves #32

### CODE REVIEW

File: envs/py311_uscuni_simplification.yml [hunk 1]
```diff
@@ -1,9 +1,9 @@
-name: py312_uscuni_simplification
+name: py311_uscuni_simplification
```
Comment (martinfleis) [inline envs/py311_uscuni_simplification.yml:1]: Can we shorten the name? It is a pain to type it to activate the env :D

Comment (jGaboardi) [reply envs/py311_uscuni_simplification.yml:1]: sure, lazy bones. any suggestions?

Comment (jGaboardi) [reply envs/py311_uscuni_simplification.yml:1]: FWIW, I create aliased functions in my shell that calls these up automatically without having to type them in.

Comment (jGaboardi) [reply envs/py311_uscuni_simplification.yml:1]: re: naming – explicit >> implicit

Comment (martinfleis) [reply envs/py311_uscuni_simplification.yml:1]: I had my called `simplification` but I can also easily override the name...

Comment (jGaboardi) [reply envs/py311_uscuni_simplification.yml:1]: That works for me.

Comment (jGaboardi) [reply envs/py311_uscuni_simplification.yml:1]: Shall I update the name of the env file and the env name itself?

Comment (martinfleis) [reply envs/py311_uscuni_simplification.yml:1]: I suppose that if we stick to a single env only, it can be an `environment.yml` in root.

Comment (jGaboardi) [reply envs/py311_uscuni_simplification.yml:1]: fair enough. I will update and re-push.

**Review by martinfleis — APPROVED** [review]
Thank you :)

**Comment by jGaboardi** [comment]
Do we want @anastassiavybornova's eyes on this or should we merge?

**Comment by martinfleis** [comment]
@anastassiavybornova We love you but I'll merge this :)

**Comment by anastassiavybornova** [comment]
@martinfleis way to go! yes to both

**Pull request merged**